app/assets/stylesheets/fae/modules/forms/_radio.scss

```diff
@@ -7,7 +7,7 @@
 }
 
 .radio_collection--vertical {
-  label {
+  .radio label {
```

Comment:
I know this is a nitpicky edit, but every [selector gets its own line](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)